test/com/nutrons/framework/test/TestEvents.java

```diff
@@ -1,34 +0,0 @@
-package com.nutrons.framework.test;
```

Comment:
You figured out why this test didn't work, right? Make sure you add one that actually tests the functionality.